README.md

```diff
@@ -1 +1,64 @@
-# saltoapis-dotnet
\ No newline at end of file
+# saltoapis-dotnet
+This repository contains the source code for the C# SALTO APIs SDK.
+
+> Some of this APIs are using gRPC. Refer to the [official gRPC documentation](https://docs.microsoft.com/en-us/aspnet/core/grpc/client?view=aspnetcore-5.0) to learn more about gRPC.
+
+
+## Authentication Example
+The SDK provides a simple gRPC interceptor that will automatically get and refresh valid access tokens and include them in all gRPC requests:
+```c#
+// create a gRPC channel
+GrpcChannel channel = GrpcChannel.ForAddress("https://nebula.saltoapis.com");
+
+// Create a new SaltoapisAuthInterceptor with your user credentials
+var authInterceptor = new SaltoapisAuthInterceptor(username, secret);
+
+// And add the interceptor to your channel
+var invoker = ChannelExtensions.Intercept(channel, authInterceptor);
+
+// Now you can use the resulting invoker to instantiate gRPC services
+var service = new UserService.UserServiceClient(invoker);
+```
+
+You can find more information about authentication on https://developer.saltonebula.com/api/authentication/.
```

Comment:
developer.saltosystems.com

Reply:
Nit: You can find more information about authentication at developer.saltosystems.com